### PR TITLE
[0.9.1]remove chunked prefill and fix ring mla mask

### PR DESCRIPTION
**Pull request by SunnyLee151064**

### CODE REVIEW

File: vllm_ascend/attention/mla_v1.py [hunk 1]
```diff
@@ -216,12 +216,17 @@ def __init__(self,
...
-        self.prefill_attn_mask = torch.triu(
+        prefill_mask = torch.triu(
             torch.ones(512,
                        512,
                        device=self.device,
-                       dtype=self.model_config.dtype),
-            1)  # 512: mask only support 512
+                       dtype=self.model_config.dtype), 1)
+        if self.model_config.dtype == torch.float16:
+            mask_value = torch.finfo(torch.float32).min
```
Comment (weijinqian0) [inline vllm_ascend/attention/mla_v1.py:225]: Why do quantitative and non-quantitative approaches differ?

**Comment by wangxiyuan** [comment]
any update?
